provider/core.rb

```diff
@@ -25,7 +25,7 @@ module Provider
   class Core
     include Compile::Core
 
-    def initialize(config, api, version_name, start_time)
```

Comment:
Maybe set some more `doc_only=false` in method headers so you don't have to specify the variable in places where it doesn't seem relevant? Also should stop the cloud build from failing